File: Atividade 1/Exercício 4.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    float valor_conta, nova_conta, porcentagem;
    printf("Digite o valor da conta: \n");
    scanf("%f",&valor_conta);
    printf("Digite a porcentagem do garcom. Numero entre 1 e 100 \n");
    scanf("%f", &porcentagem);
    nova_conta = valor_conta * (1 + (porcentagem/100));
    printf("O novo valor eh %.2f\n", nova_conta);
    return 0;
}

File: Atividade 1/Exercício 4.py
valor_conta = float(input("Digite o valor da conta: "));
while(True):
    taxa = int(input("Digite a taxa do garçom. Número inteiro entre 1 e 100: "));
    if taxa < 1 or taxa > 100:
        print("Taxa inválida, tente novamente.");
    else:
        break;
        
novo_valor = valor_conta * (1 + (taxa/100));
print(f'O novo valor da conta é {novo_valor:.2f}');